crates/terminal_view/src/terminal_view.rs

```diff
@@ -214,7 +214,13 @@ impl TerminalView {
         cx: &mut ViewContext<Self>,
     ) {
         let context_menu = ContextMenu::build(cx, |menu, _| {
-            menu.action("Clear", Box::new(Clear))
+            menu.action("New Terminal", Box::new(NewTerminal))
+                .separator()
+                .action("Copy", Box::new(Copy))
+                .action("Paste", Box::new(Paste))
+                .action("Select All", Box::new(SelectAll))
```

Comment:
This does not show any key combination as you can see on the screenshot, albeit `cmd-a` already works on mac, and I suspect that `ctrl-a` will work on Linux too.

Let's add corresponding entries into `default-macos.json` and `default-linux.json` into the
```
{
    "context": "Terminal",
...
}
```
blocks: `"cmd-a": "editor::SelectAll",` and a similar one for the Linux version.

Reply:
Taken care of works on MacOS & Linux correctly now, thanks for catching that